orthw

```diff
@@ -6,10 +6,15 @@
 required_commands="curl md5sum xz"
 required_directories="configuration_home ort_home orthw_home scancode_home exports_home"
 
-
-orthw_config_file=~/.orthwconfig
-# shellcheck source=orthconfig-template
-. "$orthw_config_file"
+orthw_config_file="$HOME/.orthwconfig"
+if [ -f "$orthw_config_file" ]; then
+  # shellcheck source=orthconfig-template
+  . "$orthw_config_file"
+else
+  echo "Missing the required configuration file '$orthw_config_file'. Please create that file, see also the default"
+  echo "configuration: https://github.com/oss-review-toolkit/orthw/blob/main/orthwconfig-template".
```

Comment:
I would also refer people to the orthw installation instructions.

```
echo "Missing the required configuration file '$orthw_config_file'. Please create this file using"
echo "https://github.com/oss-review-toolkit/orthw/blob/main/orthwconfig-template following".
echo "orthw installation guidance found in https://github.com/oss-review-toolkit/orthw#3-create-your-orthw-configuration".
```

Reply:
I incorporated parts of the idea and made it more compact to strip some redundancy.